javascript/ql/lib/semmle/javascript/Concepts.qll

```diff
@@ -84,6 +84,17 @@ abstract class FileNameSource extends DataFlow::Node { }
 abstract class DatabaseAccess extends DataFlow::Node {
   /** Gets an argument to this database access that is interpreted as a query. */
   abstract DataFlow::Node getAQueryArgument();
+
+  /** Gets a node to which a result of the access may flow. */
+  abstract DataFlow::Node getAResult();
+
+  /** Might the data returned as a JSON object like document databases do?
+   * */
```

Comment:
```suggestion
  /** 
   * Holds if the data returned is can be a user-controlled object, such as a JSON object parsed from user-controlled data. 
   * */
```
I rewrote the QLDoc. Predicates without a result should generally start with "Holds ...". 
I took some inspiration from `RemoteFlowSource::isUserControlledObject()`.

Reply:
👍